src/components/custom-field-input-text/custom-field-input-text.css

```diff
@@ -0,0 +1,84 @@
+@import '../../styles/colors.css';
+@import '../../styles/spacing.css';
+@import '../../styles/typography.css';
+
+.additional-information-icon {
+  border: 1px solid var(--palette-grey-light);
+  border-radius: 50%;
+  color: var(--palette-grey-dark);
+  cursor: pointer;
+  display: inline-block;
+  height: var(--spacing-large);
+  left: var(--spacing-medium);
+  padding-left: 1px;
+  position: relative;
+  text-align: center;
+  width: var(--spacing-large);
+}
+
+.custom-field-input-text {
+  font-family: var(--mavenlink-type-font-family);
+}
+
+.heading-container {
+  color: var(--palette-grey-dark);
+  font-size: var(--mavenlink-type-body-font-size);
+}
+
+.error .heading-container {
+  color: var(--palette-caution-base);
+}
+
+.help {
+  color: var(--palette-grey-dark);
+  font-size: var(--mavenlink-type-base-size);
+}
+
+.error .help {
+  color: var(--palette-caution-base);
+}
+
+.input-container {
+  height: 34px;
+  margin: var(--spacing-small) 0;
+}
+
+.input-container input {
+  border: 1px solid var(--palette-grey-light);
+  border-radius: var(--spacing-x-small);
+  color: var(--palette-grey-x-dark);
+  font-size: var(--mavenlink-type-base-size);
+  padding: var(--spacing-medium);
+}
+
+.input-container input::placeholder {
+  color: var(--palette-grey-dark);
+}
+
+.disabled .input-container input {
```

Comment:
Soon to come: example on using nesting versus using flat selectors.

Reply:
As an example of how overriding would currently work:

```
<CustomFieldInputText className="my_new_classname" />

.my_new_classname.disabled input {
  background-color: green;
}
```

Reply:
```
.my_new_classname input {
  composes: disabled from 'custom-field-text-input.css'
  background-color: green;
}
```

Reply:
I'm working through improving our Engineering Wiki's CSS guides with reasons for using non-nested selectors and class names. 

I think we're okay with not doing so in *this* component because:

1. This is the first release, we can iterate to improve it when we need to.
2. This is a very specific kind of component: it is just for custom fields
3. It does not have a React node as part of its properties interfaces

Though, when I do end up polishing up the new documentation, I'll try to link it here. But not a blocker!